### PR TITLE
Scale flux for conditions, differenatiate reco and true source_fov_offset

### PR DESCRIPTION
**Pull request by maxnoe**
* Implements scaling of the flux until the two other requirements are met (so we have a minimum sensitivity not a target sensitivity)

* Differentiate reco and true source_fov_offset and use the reco_source_fov_offset for estimating background rates.

With these changes, sensitivity on the cut2 files now agrees very well, sensitivity on our own cut optimization is also much better in the low energies but now shows a slightly larger disagreement at higher energies.

To improve that, I will make a second PR going to gamma/hadron efficiencies instead of cuts with the 0.8 efficiency limit that gernot uses.

### CODE REVIEW

File: pyirf/sensitivity.py [hunk 1]
```diff
@@ -104,7 +117,23 @@ def equation(relative_flux):
         )
         return np.nan
 
-    return relative_flux
+    # less then min_sigma
```
Comment (moralejo) [inline pyirf/sensitivity.py:120]: than

**Review by moralejo — APPROVED** [review]

**Review by HealthyPear — CHANGES_REQUESTED** [review]

File: examples/calculate_eventdisplay_irfs.py [hunk 2]
```diff
@@ -150,7 +153,7 @@ def main():
         gammas[gammas["selected_theta"]],
         background,
         reco_energy_bins=sensitivity_bins,
-        gh_cut_values=np.arange(-1.0, 1.005, 0.05),
+        gh_cut_values=np.arange(-0.5, 1.005, 0.01),
         theta_cuts=theta_cuts,
```
[the remark this answers is not in the document]
Comment (maxnoe) [reply examples/calculate_eventdisplay_irfs.py:157]: yes it is.  But I found that the cuts below are not used. So I started at `-0.5` to reduce the load a bit.

File: .travis.yml [hunk 3]
```diff
@@ -33,8 +33,7 @@ install:
   - sed -i -e "s/- python=.*/- python=$PYTHON_VERSION/g" environment.yml
   - conda env create -f environment.yml
   - conda activate pyirf
-  # cython needed for building gammapy
-  - pip install travis-sphinx codecov pytest-cov Cython
+  - pip install travis-sphinx
```
[the remark this answers is not in the document]
Comment (maxnoe) [reply .travis.yml:36]: yes, will fix.

File: pyirf/io/gadf.py [hunk 4]
```diff
@@ -132,6 +132,7 @@ def create_psf_table_hdu(
             "RPSF": psf.T[np.newaxis, ...].to(1 / u.sr),
         }
     )
+    print(psf['RPSF'].shape)
```
[the remark this answers is not in the document]
Comment (maxnoe) [reply pyirf/io/gadf.py:135]: yes.

**Review by HealthyPear — APPROVED** [review]

**Pull request merged**